src/pypeh/core/models/validation_dto.py

```diff
@@ -619,6 +614,7 @@ def from_peh(
     def from_observation(
         cls,
         observation: peh.Observation | pehs.Observation,
+        observable_property_id_selection: Sequence[str],
```

Comment:
same as previous comment. How does the selection differ from the keys of the observable_property_dict and why not just use a CacheView here then?